ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -724,15 +724,6 @@ struct Shape {
 
     const auto operator[](std::int64_t index) const { return this->value.without_padding()[index]; }
 
-    const auto volume() const {
```

Comment:
This is how it was called
```
tensor.get_shape().with_tile_padding().volume();
```

tensor.get_shape returns ttnn::Shape
Shape::with_tile_padding() also returns ttnn::Shape (its semantic value is - Padded Shape)
..
so for that case volume made sense and returned real physical volume
but there were also calls like this
```
    std::size_t tensor_size_bytes = input_tensor.shape().volume() * input_tensor.element_size();
```
which called volume on the ttnn::Shape (its semantic value is (Shape and Padding info)
so for this case volume returned a logical volume, which I believe **is a bug**.